<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cmd</class>
 <widget class="QMainWindow" name="cmd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="PORT_radioButton">
        <property name="text">
         <string>PORT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="PASV_radioButton">
        <property name="text">
         <string>PASV</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="mkdir_pushButton">
      <property name="text">
       <string>MKDIR</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="rmdir_pushButton">
      <property name="text">
       <string>RMDIR</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3"/>
    </item>
    <item>
     <widget class="QProgressBar" name="download_progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="upload_progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
